ChillChat.Vue/src/App.vue

```diff
@@ -1,99 +1,20 @@
 <script setup lang="ts">
+import "./assets/main.scss";
+import "./assets/styles/landing.scss";
```

Comment:
Miks tää on app.vuessa, eikä esim landingpageviewissä

Reply:
Forgot to move it. Now fixed in commit 10f7660eb3fe6eadcde42018978f1600202e0869